client/my-sites/domains/domain-management/edit/domain-types/wpcom-domain-type.jsx

```diff
@@ -0,0 +1,155 @@
+import { getDomainTypeText } from 'lib/domains';
+import { recordTracksEvent, recordGoogleEvent } from 'state/analytics/actions';
+import MaterialIcon from 'components/material-icon';
+
+class WpcomDomainType extends React.Component {
+	handleEditSiteAddressClick = () => {
+		const { domain } = this.props;
+		const domainType = getDomainTypeText( domain );
+
+		this.props.recordGoogleEvent(
+			'Domain Management',
+			`Clicked "Edit Site Address" navigation link on a ${ domainType } in Edit`,
+			'Domain Name',
+			domain.name
+		);
+
+		this.props.recordTracksEvent( 'calypso_domain_management_edit_navigation_click', {
+			action: 'edit_site_address',
+			section: domain.type,
+		} );
+	};
+
+	handleChangeSiteAddressClick = () => {
+		const { domain } = this.props;
+		const domainType = getDomainTypeText( domain );
+
+		this.props.recordGoogleEvent(
+			'Domain Management',
+			`Clicked "Change Site Address" navigation link on a ${ domainType } in Edit`,
+			'Domain Name',
+			domain.name
+		);
+
+		this.props.recordTracksEvent( 'calypso_domain_management_change_navigation_click', {
+			action: 'change_site_address',
+			section: domain.type,
+		} );
+	};
+
+	handlePickCustomDomainClick = () => {
+		const { domain } = this.props;
+		const domainType = getDomainTypeText( domain );
+
+		this.props.recordGoogleEvent(
+			'Domain Management',
+			`Clicked "Pick a custom domain" navigation link on a ${ domainType } in Edit`,
+			'Domain Name',
+			domain.name
+		);
+
+		this.props.recordTracksEvent( 'calypso_domain_management_wpcom_domain_add_domain', {
+			action: 'change_site_address',
+			section: domain.type,
+		} );
+	};
+
+	getPickCustomDomain() {
+		const { domain } = this.props;
+
+		const isWpcomDomain = get( domain, 'type' ) === domainTypes.WPCOM;
+
+		if ( ! isWpcomDomain ) {
+			return;
+		}
+
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="search"
+				text={ this.props.translate( 'Pick a custom domain' ) }
+				description={ this.props.translate( 'Register or transfer custom domain name' ) }
+			/>
+		);
+	}
+
+	getSiteAddressChange() {
+		const { domain } = this.props;
+
+		if ( domain.isWpcomStagingDomain ) {
+			return;
+		}
+
+		const isWpcomDomain = get( domain, 'type' ) === domainTypes.WPCOM;
+		const path = domainManagementChangeSiteAddress( this.props.selectedSite.slug, domain.name );
+
+		return (
+			<VerticalNavItemMulti
+				path={
+					isWpcomDomain
+						? path
+						: `https://${ this.props.domain.name }/wp-admin/index.php?page=my-blogs#blog_row_${ this.props.selectedSite.ID }`
+				}
+				external={ ! isWpcomDomain }
+				onClick={
+					isWpcomDomain ? this.handleChangeSiteAddressClick : this.handleEditSiteAddressClick
+				}
+				materialIcon="language"
+				text={
+					isWpcomDomain
+						? this.props.translate( 'Change WordPress.com Site Address' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'WordPress.com Blog/Site Address Change' )` *ES Score: 8*